bot/exts/utils/snekbox.py

```diff
@@ -279,25 +355,18 @@ async def get_code(self, message: Message) -> Optional[str]:
 
         return code
 
-    @command(name="eval", aliases=("e",))
-    @guild_only()
-    @redirect_output(
-        destination_channel=Channels.bot_commands,
-        bypass_roles=EVAL_ROLES,
-        categories=NO_EVAL_CATEGORIES,
-        channels=NO_EVAL_CHANNELS,
-        ping_user=False
-    )
-    async def eval_command(self, ctx: Context, *, code: str = None) -> None:
+    async def run_eval(
```

Comment:
I think a more generic command like run_job (and send_job for send_eval) would be better with it being expanded  and not strictly running a conventional eval

Reply:
Yeah that makes sense. Do you think it would also make more sense to say `f"Your {job_name} job has failed"`, along with the other messages, rather than just `f"Your eval job has failed"?